src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.builds

```diff
@@ -37,5 +37,24 @@
     </Project>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(__BuildType)' == 'Release'">
+    <Project Condition="'$(TargetsLinux)' == 'true' and '$(DistroName)' == 'rhel'" Include="rhel/Microsoft.NETCore.Runtime.CoreCLR-symbols.pkgproj">
```

Comment:
Ah, I didn't know that--it should be neither of those, then. The `.symbols` is part of the extension of the nupkg, not the identity. The desired names for a package and its symbol package are:

```
Microsoft.NETCore.Runtime.CoreCLR.1.0.2-foo.nupkg
Microsoft.NETCore.Runtime.CoreCLR.1.0.2-foo.symbols.nupkg
```